zuul.d/projects.yaml

```diff
@@ -1,5 +1,39 @@
 ---
+- job:
+    name: openstack-operator-tempest-multinode
+    parent: podified-multinode-edpm-deployment-crc-3comp
+    dependencies: ["openstack-k8s-operators-content-provider"]
+    vars:
+      cifmw_tempest_tempestconf_profile:
+          overrides:
+            compute.min_compute_nodes: 3
```

Comment:
I'm late, but is this really needed? python-tempestconf should be able to discover the amount of compute nodes